utils/_context/containers.py

```diff
@@ -993,6 +994,41 @@ def __init__(self, host_log_folder) -> None:
         )
 
 
+class ElasticMQContainer(TestedContainer):
+    def __init__(self, host_log_folder) -> None:
+        super().__init__(
+            image_name="softwaremill/elasticmq-native:latest",
```

Comment:
Could you pin the version ? updates of `latest` may break system-tests

Reply:
done.